src/test/java/org/sonarlint/intellij/config/global/rules/RulesTreeNodeTest.java

```diff
@@ -81,7 +81,7 @@ public void create_iterable_children() {
     parent.add(n2);
     parent.add(n3);
 
-    assertThat(parent.childrenIterable().iterator()).containsExactly(n1, n2, n3);
+    assertThat(parent.childrenIterable().iterator()).toIterable().containsExactly(n1, n2, n3);
```

Comment:
I'm surprised it can't be simplified. Have you tried: `assertThat(parent.childrenIterable()).containsExactly(n1, n2, n3);`

Reply:
Wow, I applied the change too mechanically. Good catch!

Reply:
:+1: